scripts/yaml_characters

```diff
@@ -22,8 +24,16 @@ OptionParser.new do |opts|
     excluded_locales << locale.to_sym
   end
 
+  opts.on('--exclude-path=PATH', 'Disregard characters from the given relative path') do |path|
+    excluded_paths << File.join(Dir.pwd, path, '')
+  end
```

Comment:
Technically it'd be possible to exclude telephony strings with `--exclude-key-scope=telephony`, but this might be slightly more optimized to avoid loading those files at all.